Source/Protocols/Request.swift

```diff
@@ -2,28 +2,86 @@ import Foundation
 import Argo
 import Result
 
+/**
+  Used to represent a request that doesn't return any JSON.
```

Comment:
Perhaps this should be more generic, i.e. `Used to represent a request that returns an empty response`? Since you don't _necessarily_ have to use Swish to parse JSON.